image-layout.md

```diff
@@ -29,6 +29,11 @@ The image layout is as follows:
   - It MUST be an [image index](image-index.md) JSON object.
   - See [index.json](#indexjson-file) section
 
+**Implementor's Note:**
+Additional files may be included in the directory.
+Implementations should not error when encountering unknown files.
+A common usage includes the `manifest.json` file associated with a saved docker image.
```

Comment:
Not sure if we do elsewhere, but was wondering if there's value in outlining "why" ("To allow future expansion, and to allow backward compatibility with older specifications").

Reply:
Thanks, I've added this.